wdio.conf.js

```diff
@@ -78,6 +77,7 @@ exports.config = {
     browserSpock: {
       desiredCapabilities: {
         browserName: 'firefox',
+        marionette: true,
```

Comment:
Where does this setting come from? I haven't seen this requirement before. I understood that wdio did this for us automatically with firefox browsers. I don't see this as a valid option in the docs: https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities

Reply:
I will be removing the marionette line(s) since i've found out that marionette is enabled by default
There's no easy way of telling if it is enabled by default, because it's not explicitly stated anywhere (logs nor online), but I found marionette first by seeing it was enabled by default on SauceLabs first
https://groups.google.com/d/msg/selenium-developers/d8KpXpmDp2g/ILG0hJEDBAAJ